libs/tup-components/src/tickets/TicketDetailModal/TicketReplyForm.tsx

```diff
@@ -83,7 +104,43 @@ export const TicketReplyForm: React.FC<{ ticketId: string }> = ({
               maxSize={3145728}
             />
 
+            <div className="status-wrapper">
+              <Label>Ticket Status</Label>
+              {ticketData?.Status !== 'resolved' ? (
+                <div className="status-checkbox">
+                  <Field type="checkbox" name="status" />
+                  My issue has been resolved
+                </div>
+              ) : (
+                <div className="status-checkbox">
+                  <Field type="checkbox" name="status" checked disabled /> My
+                  issue has been resolved
+                </div>
+              )}
+              {ticketData?.Status === 'resolved' ? (
+                <em>*Replying will reopen this ticket</em>
+              ) : (
+                <em>
+                  This helps us determine which users still need assistance
+                </em>
+              )}
+            </div>
+            {/* <FormikCheck 
+              name="status"
+              label="Ticket Status"
+              description="This helps us determine which users still need assistance"
+            /> */}
```

Comment:
**Commented Code Without Explanation.** Please leave note why the code is commented out, or remove it (in favor of this PR itself being the note).

Reply:
In #405, I remove it.

Reply:
Ah, indeed we did.

Since then I've accepted (even in my own PRs) that the Git and GitHub paper trail can serve as comments. I cause clutter. I'm trying to learn not to.

Please either add a comment above to explain the commented code or delete the commented code.